lib/graphql/schema_comparator/changes.rb

```diff
@@ -843,7 +843,11 @@ class FieldAdded < AbstractChange
         attr_reader :object_type, :field, :criticality
 
         def initialize(object_type, field)
-          @criticality = Changes::Criticality.non_breaking
+          @criticality = if field.graphql_name == 'id'
```

Comment:
Perhaps it should be possible to customize this?

For instance,
- Apollo's `InMemoryCache` looks for `id` or `_id` by default
- ESLint-Plugin-GraphQL's `required-fields` rule is customizable and uses `uuid` in its example

Reply:
🤔  I think customizing this might go into "linter" territory? Especially beyond the initial `id` only proposal.

Anyone consuming this library is free to do whatever they want with the output of course.